src/preamble.js

```diff
@@ -1019,7 +1019,11 @@ Module['asm'] = function(global, env, providedBuffer) {
   env['table'] = wasmTable = new WebAssembly.Table({
     'initial': {{{ getQuoted('WASM_TABLE_SIZE') }}},
 #if !ALLOW_TABLE_GROWTH
+#if WASM_BACKEND
+    'maximum': {{{ getQuoted('WASM_TABLE_SIZE') }}} + {{{ RESERVED_FUNCTION_POINTERS }}},
```

Comment:
I'd think just remove RESERVED_FUNCTION_POINTERS in WASM_BACKEND, and let WASM_TABLE_SIZE drive this. If users are setting this manually they're responsible for it anyway I'd assume.

Reply:
WASM_TABLE_SIZE some direct from wasm-emscripten-finalize.  Its not under the control of the user.   We need some way for them to say they need extra slots